config/application.rb

```diff
@@ -44,6 +44,10 @@ class Application < Rails::Application
     config.x.loadbalancer_secrets.push(ENV['LOADBALANCER_SECRET']) if ENV['LOADBALANCER_SECRET']
     config.x.loadbalancer_secrets.concat(ENV['LOADBALANCER_SECRETS'].split(':')) if ENV['LOADBALANCER_SECRETS']
 
+    # Algorithms used for calculating checksum [SHA1|SHA256|...]
+    config.x.loadbalancer_checksum_algorithm = ENV.fetch('LOADBALANCER_CHECKSUM_ALGORITHM', 'SHA1:SHA256:SHA512')
```

Comment:
Needs a documentation update in the README.md file.

Reply:
Correct